aspnet/fundamentals/dependency-injection.rst

```diff
@@ -1,10 +1,277 @@
-.. _dependency-injection:
+Dependency Injection
```

Comment:
I did add a link to AddMvc's source in the latest set of updates. If you think it would be worthwhile to speak to that (or another AddFoo method if Mvc should be in its own article) more here, we can certainly do so. The article is already pretty long, so I didn't want to grow it further. Repository in this case is meant to be a stand-in for "any application service you might have" and not something specific to data access or repositories in general. I probably can make that point more clearly.